project_task_recurring_activity/models/project_task_recurrence.py

```
@@ -0,0 +1,31 @@
+from odoo import api, fields, models
+
+
+class ProjectTaskRecurrence(models.Model):
+    _inherit = "project.task.recurrence"
+
+    old_date_recurring_task = fields.Date(
+        default=fields.Date.today(),
+    )
+
+    @api.model
+    def _get_recurring_fields(self):
```

Ruff (0.11.9)

Missing return type annotation for private function `_get_recurring_fields`

(ANN202)